src/views/recentPaymentMaster/recentPaymentMaster.tsx

```diff
@@ -0,0 +1,185 @@
+import Footer from '@/components/globals/footer/Footer';
+import Navbar from '@/components/globals/navbar/Navbar';
+import React, { useEffect, useState } from 'react';
+import { getAddressActivity, getPaymentMaster, UserOp } from '@/components/common/apiCalls/jiffyApis';
+import { Breadcrumbs, Link } from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { useRouter } from 'next/router';
+import { getFee, getTimePassed, shortenString } from '@/components/common/utils';
+import Token from '@/components/common/Token';
+import { NETWORK_ICON_MAP } from '@/components/common/constants';
+import Skeleton from 'react-loading-skeleton-2';
+
+export const BUTTON_LIST = [
+    {
+        name: 'Default View',
+        key: 'Default View',
+    },
+    {
+        name: 'Original',
+        key: 'Original',
+    },
+];
+const columns = [
+    { name: 'Hash', sort: true },
+    { name: 'Age', sort: true },
+    { name: 'Sender', sort: true },
+    { name: 'Target', sort: true },
+    { name: 'Fee', sort: true },
+];
+function RecentPaymentMaster(props: any) {
+    const router = useRouter();
+    const [tableLoading, setTableLoading] = useState(true);
+    const hash = props.slug && props.slug[0];
+    const network = router.query && router.query.network;
+
+    const [useOpsData, setuserOpsData] = useState<UserOp[]>();
+
+    const refreshUserOpsTable = async (name: string, network: string) => {
+        setTableLoading(true);
+        const userops = await getPaymentMaster(name || '0xf4419be44d713b62eb344531662d173e20fe584f', network || 'optimism-goerli');
```

Comment:
do change the function name to getPaymasterDetails(....) 

and why are we passing a default value. If there is no name , we should show and error message.